diff --git a/dist/source/utils/install-dependencies.js b/dist/source/utils/install-dependencies.js
index a323783a587216290ce6537adc9347b8c390b0f2..b609375f42f753d55ef787e8e08095361590f025 100644
--- a/dist/source/utils/install-dependencies.js
+++ b/dist/source/utils/install-dependencies.js
@@ -11,6 +11,11 @@ var log_symbols_1 = __importDefault(require("log-symbols"));
 var constants_1 = require("./constants");
 var close_ora_on_sigint_1 = require("./close-ora-on-sigint");
 var installDependencies = function (packageManager) {
+    if(packageManager === 'pnpm') {
+        console.warn(`\n🚨🚨🚨🚨🚨🚨\nUsing \`npm\` instead of \`${packageManager}\` to install react-email preview app dependencies\nThis patch should be removed when react-email can support existing in a pnpm workspace.\nsee: https://github.com/resendlabs/react-email/issues/881\n🚨🚨🚨🚨🚨🚨\n`);
+        packageManager = 'npm';
+    }
+
     var spinner = (0, ora_1.default)('Installing dependencies...\n').start();
     (0, close_ora_on_sigint_1.closeOraOnSIGNIT)(spinner);
     shelljs_1.default.cd(node_path_1.default.join(constants_1.REACT_EMAIL_ROOT));
